catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -163,6 +165,7 @@ public DynamicSchemaResolver(List<String> additionalFields) {
     ConfigurationStore.getInstance().addConfigurationListener(this);
     this.schemaFields = new SchemaFields();
     metadataMaximumBytes = getMetadataSizeLimit();
+    caseInsensitiveSort = "true".equals(System.getProperty("solr.sort.case-insensitive"));
```

Comment:
✏️ There is one `solr.query.` system property and I am working on a PR to add another one.  Might consider making this `solr.query.sort.caseInsensitive`.

https://github.com/codice/ddf/blob/b50f3c254f29510bd2999d1319bd26fc979b674f/catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java#L155